### ISSUE TITLE
How do I setup this lsp in nvim?

### ISSUE THREAD

**Issue by hbarua05** [processing-language-server/LS4P#8]
Could you kindly help me set this lsp for nvim as I would like to work in processing in the nvim editor? Thanks!

**Comment by dantearaujo1** [comment]
i would appreciate too!

**Comment by kresdjan** [comment]
Yeah an npm package would be great, so other editor can install this too...

**Comment by pr1metine** [comment]
Hmm I'm working on it. Did anyone succeed in getting this server up and running? I'm failing on that...

**Comment by dantearaujo1** [comment]
Any news?

**Comment by tiagohermano** [comment]
I'm also interested.

**Comment by nicouhoh** [comment]
I would appreciate this as well!
